### PR TITLE
[tests-only][full-ci] Remove ocis/reva related scenario tags

### PR DESCRIPTION
**Pull request by roshanlc**
## Description
This is one of the two PRs which removes `issue-ocis-*` tags from the tests.

The follow-up PR is at https://github.com/owncloud/core/pull/40622

## Related Issue

- Related to 
https://github.com/owncloud/core/issues/40572#

## How Has This Been Tested?

- CI

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 
- [ ] Changelog item, see [TEMPLATE](https://github.com/owncloud/core/blob/master/changelog/TEMPLATE)

### CODE REVIEW

File: tests/acceptance/features/apiAuthWebDav/webDavMKCOLAuth.feature [hunk 1]
```diff
@@ -30,7 +30,7 @@ Feature: create folder using MKCOL
       | /remote.php/dav/files/%username%/PARENT/parent.txt |
     Then the HTTP status code of responses on all endpoints should be "401"
 
-  @skipOnOcV10 @issue-ocis-5049 @issue-ocis-reva-9 @issue-ocis-reva-197
+  @skipOnOcV10 @issue-ocis-reva-197
```
Comment (grgprarup) [inline tests/acceptance/features/apiAuthWebDav/webDavMKCOLAuth.feature:33]:
```suggestion
  @skipOnOcV10
```

File: tests/acceptance/features/apiAuthWebDav/webDavMKCOLAuth.feature [hunk 2]
```diff
@@ -41,7 +41,7 @@ Feature: create folder using MKCOL
       | /remote.php/dav/files/%username%/PARENT/parent.txt |
     Then the HTTP status code of responses on all endpoints should be "404"
 
-  @skipOnOcV10 @issue-ocis-5049 @issue-ocis-reva-9 @issue-ocis-reva-197
+  @skipOnOcV10 @issue-ocis-reva-197
```
Comment (grgprarup) [inline tests/acceptance/features/apiAuthWebDav/webDavMKCOLAuth.feature:44]:
```suggestion
  @skipOnOcV10
```

File: tests/acceptance/features/apiAuthWebDav/webDavPROPPATCHAuth.feature [hunk 3]
```diff
@@ -34,7 +34,7 @@ Feature: PROPPATCH file/folder
       | /remote.php/dav/files/%username%/PARENT/parent.txt |
     Then the HTTP status code of responses on all endpoints should be "401"
 
-  @issue-ocis-reva-9 @issue-ocis-reva-197
+  @issue-ocis-reva-197
```
Comment (grgprarup) [inline tests/acceptance/features/apiAuthWebDav/webDavPROPPATCHAuth.feature:37]:
```suggestion
```

File: tests/acceptance/features/apiMain/checksums.feature [hunk 4]
```diff
@@ -170,7 +170,7 @@ Feature: checksums
       | dav_version | checksum                                                                                            |
       | new         | SHA1:ce5582148c6f0c1282335b87df5ed4be4b781399 MD5:56e57920c3c8c727bfe7a5288cdf61c4 ADLER32:1048035a |
 
-  @issue-ocis-reva-56 @newChunking @issue-ocis-1321
+  @newChunking @issue-ocis-1321
```
Comment (grgprarup) [inline tests/acceptance/features/apiMain/checksums.feature:173]:
```suggestion
  @newChunking
```

File: tests/acceptance/features/apiAuthWebDav/webDavMKCOLAuth.feature (hunk 2 above)
Comment (roshanlc) [reply tests/acceptance/features/apiAuthWebDav/webDavMKCOLAuth.feature:44]: All remaining `issue-ocis-*` tags are removed in follow-up PR https://github.com/owncloud/core/pull/40622

**Review by phil-davis — APPROVED** [review]

**Pull request merged**